.prettierignore

```diff
@@ -1 +0,0 @@
-contracts/
```

Comment:
Is there a reason why solidity contracts are ignored for prettier? 

maybe it didn't work well enough back when the repo was created? 

Having a consistent formatting across the code base seems valuable

Reply:
yeah it's a legacy thing we inherited from Set labs. Adding consistent formatting using prettier sgtm